35. Search Insert Position.py

```diff
@@ -0,0 +1,3 @@
+class Solution(object):
+    def searchInsert(self, nums, target):
+             return len([x for x in nums if x<target])

```

Comment:
No list comprehension. In fact, no most of the pythonic idioms. Pythonic idioms hides complexity and does not scale. It is critical you see complexity explicitly so you can think in complexity.